ctru-rs/examples/mii-selector.rs

```diff
@@ -0,0 +1,45 @@
+use ctru::applets::mii_selector::MiiSelector;
+use ctru::prelude::*;
+
+fn main() {
+    ctru::init();
+
+    let gfx = Gfx::init().expect("Couldn't obtain GFX controller");
+    let hid = Hid::init().expect("Couldn't obtain HID controller");
+    let apt = Apt::init().expect("Couldn't obtain APT controller");
+    let _console = Console::init(gfx.top_screen.borrow_mut());
+
+    let mut mii_selector = MiiSelector::init();
+    mii_selector.set_initial_index(3);
+    mii_selector.blacklist_user_mii(0.into());
+    mii_selector.set_title("Great Mii Selector!");
+
+    let result = mii_selector.launch();
+
+    println!("\x1b[0;0HIs Mii selected?: {:?}", result.is_mii_selected);
+    println!("\x1b[1;0HValid checksum?: {:?}", result.valid_checksum());
+    println!("\x1b[2;0HMii type: {:?}", result.mii_type);
+    println!("\x1b[3;0HMii checksum: {:?}", result.checksum);
+    println!("\x1b[4;0HName: {:?}", result.name());
+    println!("\x1b[5;0HAuthor: {:?}", result.author());
+    println!(
+        "\x1b[5;0HDoes it have moles?: {:?}",
```

Comment:
This message and the "author" message are both on row 5. In this order the "Moles" message wipes the "Author" one.